elide-contrib/elide-swagger/testng.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I agree. Best to avoid these `testng.xml` files unless there is a good reason for needing them. In the past these have led to problems of not getting updated and new tests are then not consistently run :(